src/main/java/com/zendesk/maxwell/schema/Table.java

```diff
@@ -259,7 +258,10 @@ public String getPKString() {
 			return null;
 	}
 
+	@JsonProperty("primary-key")
 	public void setPKList(List<String> pkColumnNames) {
-		this.pkColumnNames = pkColumnNames;
+		this.pkColumnNames = new ArrayList<>();
```

Comment:
Check out https://docs.oracle.com/javase/tutorial/java/generics/genTypeInference.html, especially "Type Inference and Instantiation of Generic Classes"